qcodes/instrument_drivers/rigol/DS1074Z.py

```diff
@@ -0,0 +1,207 @@
+from typing import Union
+import numpy as np
+
+from qcodes import VisaInstrument
+from qcodes import InstrumentChannel
+from qcodes import Instrument
+from qcodes import ChannelList
+from qcodes import ParameterWithSetpoints
+from qcodes.utils.validators import Numbers, Enum
+from qcodes.utils.validators import Arrays
+
+
+class RigolDS1074ZChannel(InstrumentChannel):
+    """
+    Contains methods and attributes specific to the Rigol
+    oscilloscope channels.
+
+    The output trace from each channel of the oscilloscope
+    can be obtained using 'trace' parameter.
+    """
+
+    def __init__(self,
+                 parent: Union[Instrument, 'InstrumentChannel'],
+                 name: str,
+                 channel
+                 ):
+        super().__init__(parent, name)
+        self.channel = channel
+
+        self.add_parameter("vertical_scale",
+                           get_cmd=":CHANnel{}:SCALe?".format(channel),
+                           set_cmd=":CHANnel{}:SCALe {}".format(channel, "{}"),
+                           get_parser=float
+                           )
+
+        self.add_parameter("trace",
```

Comment:
please set the snapshot_value to False for this parameter

Reply:
Done